### PR TITLE
JBPM-5990 - Fix GWT compilation issue, use GWT messages instead of Er…

### PR DESCRIPTION
**Pull request by cristianonicolai**
…rai translation

### CODE REVIEW

**Comment by jhrcek** [comment]
Jenkins execute full downstream build

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AdvancedSearchFiltersViewImpl.java [hunk 1]
```diff
@@ -220,11 +224,11 @@ protected void onDateValueChange(final String label,
         final DateRange dateRange = new DateRange(fromDateValue,
                                                   toDateValue);
         addActiveFilter(label,
-                        translationService.format(Constants.FROM)
+                        constants.From()
```
Comment (cristianonicolai) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AdvancedSearchFiltersViewImpl.java]: @rbarriuso FYI, somehow GWT compiler did not like mixing both GWT and Errai i18n in the same interface.

Comment (rbarriuso) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AdvancedSearchFiltersViewImpl.java]: Yeah, I saw it, it's strange. Thanks for the info!

**Pull request merged**